cms/home/migrations/0002_create_homepage.py

```diff
@@ -37,4 +37,4 @@ class Migration(migrations.Migration):
 
     dependencies = [("home", "0001_initial")]
 
-    operations = [migrations.RunPython(create_homepage)]
+    operations = [migrations.RunPython(create_homepage, migrations.RunPython.noop)]
```

Comment:
We should revert this; any migrations that have already merged should not be modified.

Reply:
`migrations.RunPython.noop` returns `None`, which is the equivalent of not specifying it at all as the `reverse_code` kwarg defaults to `None`.

This makes the linter happy